tutorial/4. multiple_grid_cells_and_hypercube.ipynb

```diff
@@ -0,0 +1,1231 @@
+       "4                   39.42402937477069       8.36348064183748   \n",
+       "...                               ...                    ...   \n",
+       "10095               39.42402937477069  9.845808746366072e-12   \n",
+       "10096               39.42402937477069  7.087406982477918e-11   \n",
+       "10097               39.42402937477069 2.7535987828509794e-10   \n",
+       "10098               39.42402937477069 1.7562622112762792e-10   \n",
+       "10099               39.42402937477069  2.382562360546678e-10   \n",
+       "\n",
+       "              CONC.B.mol m-3     CONC.C.mol m-3  \n",
+       "0          5.189847361451441  8.232724658048902  \n",
+       "1          7.627884156099922  7.755704074280443  \n",
+       "2          1.800810512897043 3.1521303116276513  \n",
+       "3          6.395686755662961  2.992571907848492  \n",
+       "4          5.444168463251003 2.5704255964858658  \n",
+       "...                      ...                ...  \n",
+       "10095  2.882668010602939e-10 1.8484674768622245  \n",
+       "10096 1.9636321793712178e-09  9.173684129940602  \n",
+       "10097 6.9122126355250565e-09 12.816928273762235  \n",
+       "10098  4.327778790968643e-09  8.449758225729914  \n",
+       "10099  5.773959484580879e-09 16.434735167020584  \n",
+       "\n",
+       "[10100 rows x 7 columns]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "concentrations_solved_expanded = []\n",
+    "time = []\n",
+    "for i in range(0, sim_length + 1, time_step_length):\n",
+    "    for j in range(0, num_grid_cells):\n",
+    "        concentrations_solved_expanded.append({key: value[j] for key, value in concentrations_solved[int(i/time_step_length)].items()})\n",
+    "        time.append(i)\n",
+    "df_expanded = pd.DataFrame(concentrations_solved_expanded)\n",
+    "df_expanded = df_expanded.rename(columns = {'A' : 'CONC.A.mol m-3', 'B' : 'CONC.B.mol m-3', 'C' : 'CONC.C.mol m-3'})\n",
+    "df_expanded['time.s'] = time\n",
+    "df_expanded['ENV.temperature.K'] = np.repeat(temperatures[0], (sim_length/time_step_length + 1.0) * num_grid_cells)\n",
+    "df_expanded['ENV.pressure.Pa'] = np.repeat(pressures[0], (sim_length/time_step_length + 1.0) * num_grid_cells)\n",
+    "df_expanded['ENV.air number density.mol m-3'] = np.repeat(state.get_conditions()['air_density'][0], (sim_length/time_step_length + 1.0) * num_grid_cells)\n",
+    "df_expanded = df_expanded[['time.s', 'ENV.temperature.K', 'ENV.pressure.Pa', 'ENV.air number density.mol m-3', 'CONC.A.mol m-3', 'CONC.B.mol m-3', 'CONC.C.mol m-3']]\n",
+    "display(df_expanded)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "81890f02",
+   "metadata": {},
+   "source": [
+    "## 5. Visualizing the Results\n",
+    "Lastly, the three species are plotted out with Seaborn's <i>lineplot()</i> function, which takes in an input x and y variable with the confidence interval (CI) displayed.</br>\n",
```

Comment:
how do you set the CI interval or is it default to `p=0.05`? What if I want to add max/min lines rather than the CI lines?

Reply:
Sounds good!